time stamp
geometry_msgs/Twist twist
float32 gripper
